src/protocols/bdx/BdxTransferSession.cpp

```diff
@@ -948,6 +948,12 @@ const char * TransferSession::OutputEvent::ToString(OutputEventType outputEventT
     }
 }
 
+const char * TransferSession::OutputEvent::TypeToString(const OutputEventType outputEventType)
+{
+    OutputEvent event(outputEventType);
```

Comment:
This is a really weird way of doing it.  Wouldn't it make more sense to have something like:

```
const char * TransferSession::OutputEvent::TypeToString(OutputEventType outputEventType)
{
    switch (....) ....
}

const char * TransferSession::OutputEvent::ToString(OutputEventType outputEventType)
{
    return TypeToString(outputEventType);
}
```

(i.e. have the member function call the static function, which it can always do), instead of going the other way and needing to create fake objects?

Reply:
done :)